pybleau/app/model/dataframe_plot_manager.py

```diff
@@ -102,6 +103,9 @@ class DataFramePlotManager(DataElement):
     containers_in_use = Property(Set,
                                  depends_on="contained_plots:container_idx")
 
+    #: Plot template manager
+    template_manager = Instance(PlotTemplateManager)
+
```

Comment:
The intention was that a `DataframePlotManager` is created with a `template_interactor` keyword argument by the consuming/downstream application. That's the primary entry point. Maybe we can discuss how it might be changed?

Reply:
Discussed offline and resolved with @jonathanrocher